lib/main.coffee

```diff
@@ -135,6 +135,8 @@ module.exports =
         if confFile then execOptions.cwd = path.dirname(confFile)
         return helpers.exec(command, parameters, execOptions).then (result) =>
           try
+            # Strip time+memory information at the end; workaround for https://github.com/squizlabs/PHP_CodeSniffer/issues/969
```

Comment:
Can you add a `# FIXME: Remove when the above issue is fixed` (Or below, I don't really care which).

Just to make it explicit that this is something that should be removed as soon as we can do so.